docs/build/eps.md

```diff
@@ -720,3 +720,50 @@ Once you have all the necessary components setup, follow the instructions to [cr
 
 * Replace `--minimal_build` with `--minimal_build extended` to enable support for execution providers that dynamically create kernels at runtime, which is required by the CoreML EP.
 * Add `--use_coreml` to include the CoreML EP in the build
+
+## XNNPACK
+
+Usage of XNNPACK on Android/Web/Windows/Linux platforms is via the XNNPACK EP.
+
+See the [XNNPACK Execution Provider](../execution-providers/Xnnpack-ExecutionProvider.md) documentation for more details.
+
+The pre-built ONNX Runtime Mobile package for Android doesn't include the XNNPACK EP.
+
```

Comment:
May be simpler to say that only the onnxruntime-android package includes so that iOS is covered as well.

Reply:
Would also be good to add a note saying we don't support XNNPACK on iOS currently.

Reply:
Okay, I will add a note here and will update it when iOS build-stuff is finished.

Reply:
Sorry - I think the change adds confusion as it mentions both 'full' and 'mobile' without the package name. 'The pre-build onnxruntime-android package for Android includes the XNNPACK EP.' avoids that and directly gives the user the exact package name to look for.

nit: Capitalize the 'we' at the start of "we don't support ...".

Reply:
Fixed, thanks.